main.tf

```diff
@@ -9,6 +9,7 @@ resource "aws_instance" "webserver" {
   tags = {
     Name = "webserver"
   }
+  ebs_optimized = true
 }
```

Comment:
**Accurics remediation**
 
**[LOW]: Ensure EBS optimization is enabled for AWS Instance**
 
**Vulnerability**
AWS Instances are not EBS optimized. Enabling EBS optimization ensures optimum performance.
 
**Remediation**
An Amazon EBS–optimized instance uses an optimized configuration stack and provides additional, dedicated capacity for Amazon EBS I/O. This optimization provides the best performance for your EBS volumes by minimizing contention between Amazon EBS I/O and other traffic from your instance.

source: accurics-localhost_harshit